static-site/content/community-datasets.yaml

```diff
@@ -200,3 +200,35 @@ data:
     maintainers: Center for Medical Genetics, Keio University School of Medicine
     date: '2022-03-23'
     title: Phylogenetic analysis of SARS-CoV-2 diversity in Japan
+  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/L
+    maintainers: Jin-Won Song (Korea University)
+    date: '2022-04-23'
+    title: Hantavirus L segment
+  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/M
+    maintainers: Jin-Won Song (Korea University)
+    date: '2022-04-23'
+    title: Hantavirus M segment
+  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/S
+    maintainers: Jin-Won Song (Korea University)
+    date: '2022-04-23'
+    title: Hantavirus S segment
+  - url: https://nextstrain.org/community/KU-MolecularVirology/Influenzavirus/HA
+    maintainers: Man-Seong Park (Korea University)
+    date: '2022-04-23'
+    title: INFLUENZA HA
+  - url: https://nextstrain.org/community/KU-MolecularVirology/Influenzavirus/NA
+    maintainers: Man-Seong Park (Korea University)
+    date: '2022-04-23'
+    title: INFLUENZA NA
+  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/L
+    maintainers: Nam-Hyuk Cho (Seoul National University)
+    date: '2022-04-23'
+    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) L segment
+  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/M
+    maintainers: Nam-Hyuk Cho (Seoul National University)
+    date: '2022-04-23'
+    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) M segment
+  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/S
+    maintainers: Nam-Hyuk Cho (Seoul National University)
+    date: '2022-04-23'
+    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) S segment
```

Comment:
Hi @KU-MolecularVirology,

Thank you for sending your request via GitHub. Instead of adding new lines at the end of the file, could you replace these existing lines?

https://github.com/nextstrain/nextstrain.org/blob/ff13cb7bc72c35a94d0cf04a12f527a3ec75f28e/static-site/content/community-datasets.yaml#L59-L90

Otherwise, we have duplicate entries for your datasets:

![duplicate-community-datasets](https://user-images.githubusercontent.com/13424970/165147741-d1c007bf-e53b-44e5-8bf3-2019907c358d.png)

Reply:
Thanks your comment.  

I apologize for the confusion.

We want to replace URL from previous version to given URL.
We will re-upload  new URL of the dataset to change the URL from previous version (updated date 2022-03-03).

As soon as possible, we will update final repositories.

We really appreciate your great supports.

Sincerely,

Reply:
Dear Nextstrain team,
I sent a comment on behalf of Professor Jin-Won Song (Korea University) and Nam-Hyuk Cho (Seoul National University)from the Republic of Korea (ROK).

I am a researcher at Korea University in the ROK.

Initially, we contact Dr. Trevor Bedford, and we have prepared three community sites (about three different viruses; swine influenza virus, SFTSV, and hantavirus) that would be fit in the web of Nextstrain Community.

First dataset (git owner: modoree) was posted on Nextstrain community page previously. However, we thought that the URL is not clear for the description of virus characterization and contributed groups. Thus, we want to replace the URL (database of the hantavirus, influenzavirus, and SFTSV).

I provided the new URL submitted to Github.
1) Hantavirus
https://nextstrain.org/community/KU-MV/Hantavirus/L
https://nextstrain.org/community/KU-MV/Hantavirus/M
https://nextstrain.org/community/KU-MV/Hantavirus/S
2) Influenzavirus
https://nextstrain.org/community/KU-MV/Influenzavirus/HA 
https://nextstrain.org/community/KU-MV/Influenzavirus/NA
3) SFTSV
https://nextstrain.org/community/KU-MV/SFTSV/L
https://nextstrain.org/community/KU-MV/SFTSV/M
https://nextstrain.org/community/KU-MV/SFTSV/S

We really appreciate your great supports.  

As soon as, we will remove the previous URL (supported by modoree).
Would you please let us know what we have to do for this? 

I would appreciate it if you could replace the URL and post our dataset on the front page (or community page) on the Nextstrain web page.

Please let me know the due date of the process for posting our datasets and give me a reply if you have any questions.

Thank you so much for your kind consideration. We are looking forward to hearing from you soon.

Sincerely,

Reply:
Hello @KU-MV,

I understand that you want to replace the `modoree` URLs to use your new GitHub user `KU-MV`. I have updated this Pull Request with the necessary changes: 4a1e4a1273a9c50e794e82d95739fbef125d0aab

Unfortunately, we are unable to provide a preview website for you to confirm the changes. However, here is a screenshot when I test it on my computer:

<img width="500" alt="KU-MV dataset listing preview" src="https://user-images.githubusercontent.com/13424970/165213888-0d40230a-ba25-44de-a805-2dcb8987e713.png">

Let me know if this looks good, and we can make these changes on [nextstrain.org/community](https://nextstrain.org/community/).